index.html

```diff
@@ -196,13 +194,13 @@ <h1 class="players-name"></h1>
             <div class="about-section-header-div">
               <header class="about-section-header">
                 <span class="Welcome-to">Welcome to:</span>
-                <span class="ICC-Mens">ICC Men's T20 world cup Australia,2022</span>
+                <span class="ICC-Mens">ICC Men's T20 world cup,2024</span>
               </header>
             </div>
             <div class="about-info-div">
               <p class="about-info">
-                The ICC announced that the winning side at the ICC Men's T20 World Cup 2022 in Melbourne on November 13 would receive
-                prize money of $1.6 million i.e about 13 crores
+                The ICC has announced the prize money for the upcoming men's T20 World Cup. The title winners this season will take home a whopping US$1.6 million while the runners up will win
+                half the amount of the title winners' prize, which is $800,000.
               </p>
```

Comment:
- [ ] Please make sure the about page's hero section's paragraph description is centered on the page and matches the design guidelines please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210879536-1cfad613-7917-472c-b2a3-933c4f56c7e6.png)
how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210879645-910f794e-0410-4cf4-977c-bd7a8e591636.png)

- [ ] [OPTIONAL] I think it will be better if your hero section in the about page and also in the home page take the full viewport height, as it is in the design guidelines, please see how this section is taking the full viewport's height :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210879977-14389ac1-ac69-435f-b258-0ae9b0d808f5.png)